Formula/s/samba.rb

```diff
@@ -56,7 +56,10 @@ class Samba < Formula
   end
 
   conflicts_with "jena", because: "both install `tdbbackup` binaries"
+  conflicts_with "ldb",
+    because: "both install `ldbadd`, `ldbdel`, `ldbedit`, `ldbmodify`, `ldbrename`, `ldbsearch` binaries"
   conflicts_with "puzzles", because: "both install `net` binaries"
+  conflicts_with "tdb", because: "both install `tdbrestore`, `tdbtool` binaries"
```

Comment:
`samba` uses `tdb` and `ldb`.

Reply:
Ah, good catch!
I had this code in my branch for too long, that it got fixed in https://github.com/Homebrew/homebrew-core/pull/181555 🤦